<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select</class>
 <widget class="QDialog" name="select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>企业人事管理系统</string>
  </property>
  <widget class="QPushButton" name="selectBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="selectLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="dispalyGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>511</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>搜索结果</string>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="jobNumLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>工号</string>
    </property>
   </widget>
   <widget class="QLabel" name="connLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>联系方式</string>
    </property>
   </widget>
   <widget class="QLabel" name="posiLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>职位</string>
    </property>
   </widget>
   <widget class="QLabel" name="departLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>部门</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="jobNumLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="connLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="posiLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="departLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
